javascript/.env.local.SAMPLE

```diff
@@ -1,4 +1,3 @@
 REACT_APP_AUTH0_DOMAIN=FILL-IT-IN
 REACT_APP_AUTH0_AUDIENCE=https://FILL-IT-IN.herokuapp.com
-REACT_APP_AUTH0_CLIENT_ID=FILL-IT-IN
-
+REACT_APP_AUTH0_CLIENT_ID=FILL-IT-IN
```

Comment:
Since there aren't any actual code changes (it looks like it's just random whitespace), can we undo the changes in the SAMPLE files entirely so they don't clutter up the diff?